_docs-sources/iac/getting-started/deploying-a-module.md

```diff
@@ -1,27 +1,267 @@
 
-Tortor at risus viverra adipiscing. In hac habitasse platea dictumst quisque sagittis purus sit amet. Pretium viverra suspendisse potenti nullam ac tortor. Nulla facilisi etiam dignissim diam quis enim. Massa massa ultricies mi quis hendrerit dolor magna eget est. Tincidunt tortor aliquam nulla facilisi cras fermentum. Sollicitudin nibh sit amet commodo nulla facilisi nullam vehicula ipsum. Faucibus purus in massa tempor nec feugiat nisl pretium. Suspendisse sed nisi lacus sed viverra tellus in hac habitasse. Fermentum leo vel orci porta non pulvinar neque laoreet suspendisse. At auctor urna nunc id cursus metus aliquam eleifend mi. Condimentum mattis pellentesque id nibh tortor. Enim neque volutpat ac tincidunt vitae semper quis. Lobortis feugiat vivamus at augue eget arcu dictum varius duis. Tellus in metus vulputate eu scelerisque felis imperdiet. Sed arcu non odio euismod lacinia at quis.
+This module could be referenced many times to create any number of AWS Lambdas and IAM roles.
 
-## Curl the endpoint
 
-Congue quisque egestas diam in arcu. Faucibus a pellentesque sit amet porttitor. Placerat in egestas erat imperdiet sed euismod. Duis convallis convallis tellus id interdum velit. Congue eu consequat ac felis. Eu nisl nunc mi ipsum faucibus vitae aliquet. In hendrerit gravida rutrum quisque non. Dui ut ornare lectus sit. Phasellus vestibulum lorem sed risus ultricies tristique nulla aliquet enim. At imperdiet dui accumsan sit amet nulla facilisi morbi tempus. Viverra aliquet eget sit amet tellus cras adipiscing. Dolor sit amet consectetur adipiscing elit duis tristique sollicitudin. Ut pharetra sit amet aliquam id diam maecenas ultricies mi. Nullam vehicula ipsum a arcu cursus.
+### Create a basic file structure
+First, create the directories and files that will contain the Terraform configuration.
 
-## Tear it down
+```sh
+mkdir -p terraform-aws-gw-lambda-tutorial/modules/lambda
+touch terraform-aws-gw-lambda-tutorial/modules/lambda/main.tf
+touch terraform-aws-gw-lambda-tutorial/modules/lambda/variables.tf
+touch terraform-aws-gw-lambda-tutorial/modules/lambda/outputs.tf
+```
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+### Define the module resources
 
-## You did it!
+First, define the resources that should be created by the module. This is where you define resource level blocks provided by Terraform. For this module, we need an AWS Lambda and an AWS IAM role that will be used by the Lambda.
```

Comment:
```suggestion
First, define the resources that should be created by the module. This is where you define resource level blocks provided by Terraform. For this module, we need an AWS Lambda function and an AWS IAM role that will be used by the Lambda function.
```

There's a number of instances in this guide where we are referencing deploying `AWS Lambda`, where it feels like we should be referring to the function rather than the service itself, like we are with `AWS IAM role`. For example the snippet below we are deploying an `aws_lambda_function` resource and it feels like we should be referring to it as an `AWS Lambda Function`. 

I'm reading it to myself and it sounds odd just as `AWS Lambda` (but that also could just be me)